.github/CODEOWNERS

```
@@ -15,5 +15,5 @@ po/pt-BR.po @rastringer @hugojacob @henrif75 @joaovicmendes @jcvicelli @azevedoa
 po/ru.po @istolga @baltuky @zvonden @dyeroshenko
 po/tr.po @duyguisler @enes1313
 po/uk.po @dyeroshenko
-po/zh-CN.po @suetfei @wnghl @anlunx @kongy @noahdragon @superwhd @emmali01 @SketchK @candysonya @AgainstEntropy
+po/zh-CN.po @suetfei @wnghl @anlunx @kongy @noahdragon @superwhd @emmali01 @SketchK @candysonya @AgainstEntropy @illustager
```

Please move this to a separate change.

---

Let's just take it out here:

```suggestion
po/zh-CN.po @suetfei @wnghl @anlunx @kongy @noahdragon @superwhd @emmali01 @SketchK @candysonya @AgainstEntropy
```